sequence_processing_pipeline/NuQCJob.py

```diff
@@ -499,3 +574,26 @@ def _generate_job_script(self, max_bucket_size):
                                     pmls_path=self.pmls_path))
 
         return job_script_path
+
+    def parse_logs(self):
+        log_path = join(self.output_path, 'logs')
+        files = sorted(glob(join(log_path, '*')))
+        msgs = []
+
+        # assume that the only possible files in logs directory are '.out'
+        # files and zero, one, or many 'seqs.movi.n.txt.gz' files.
```

Comment:
Why not have the glob express these assumptions?

Reply:
Worth revisiting at some point: https://github.com/biocore/mg-scripts/issues/166